src/Package/TestApp/Files/VariableGrid02.tree

```diff
@@ -520,7 +520,7 @@
                                       "Properties": [
                                         {
                                           "Name": "Text",
-                                          "Value": "V1"
+                                          "Value": "[,1]"
```

Comment:
We were showing `V1` because that is an auto-generated column name when you convert matrix to data frame. I changed the code to avoid that conversion completely now, so for matrices, there are no auto-generated names anymore, and the code on C# side that generates columns kicks in (same one that produces them for 1D vectors). I prefer this approach, because it's now clear from column names whether they were originally named or not.